### ISSUE TITLE
[Annoying] Web interface lies about paths being relative to /web/

### ISSUE THREAD

**Issue by stephen304** [oenvoyage/brender#24]
The brender web interface claims that the project paths are relative to the web folder, while they are actually relative to the brender main folder. Have you ever used relative paths with projects and had it work successfully? If not, I would like to go ahead and modify the web interface to use paths relative to brender.

This problem affects the "server test" feature and job creation because the web interface works with paths relative to `web`, but after the job is created, the paths have to be changed relative to brender root in order for the render to complete successfully.

This is really annoying, since I have to change blend path to `../blend` to create the job, then switch it back to `blend` so that blender can find the blend files.

**Comment by stephen304** [comment]
Just pasting this here so I can change it later. Git confuses me sometimes, Idk where it will go if I commit it somewhere and I'm not sure where to commit it to. After you merge my pull request I will commit this.

Settings.php Line 321
From: `$test_path=get_path($name,"blend",$server_os);`
To: `$test_path= "../" . get_path($name,"blend",$server_os);`

Functions.php Line 521
From: `$blend_path =get_path($project,"blend",$server_os);`
To: `$blend_path = "../" . get_path($project,"blend",$server_os);`

Functions.php Line 551
From: `$scenes_path =get_path($project,"blend",$server_os);`
To: `$scenes_path = "../" . get_path($project,"blend",$server_os);`

**Comment by oenvoyage** [comment]
Yes it got broken a while ago... i do mostly use absolute paths... 
but a quick look at your changes, i would not do this :
Functions.php Line 521
From: $blend_path =get_path($project,"blend",$server_os);
To: $blend_path = "../" . get_path($project,"blend",$server_os);

because it will break the absolute path , no? 

the same goes for  settings.php line 321 . If you add "../". it breaks if u use abolute paths... 
maybe add a check to see if the path is absolute... 
not sure what is ideal solution.

**Comment by stephen304** [comment]
Don't absolute paths on mac and linux start with a forward slash? Absolute paths on windows will have `:` as the second character, and all relative paths should start with a character and not have `:` as the second character since they are not allowed in file/folder names.

**Comment by fsiddi** [comment]
Was this issue solved in the end? Can I close it?

**Comment by oenvoyage** [comment]
I would say no... paths in general are still a bit messy... and need some maintenance and cleaning

**Comment by fsiddi** [comment]
All right, we schedule this for 1.0 release in the future :)
